### PR TITLE
ref(nextjs): Improve tunnel example

### PR DESCRIPTION
**Pull request by fifi98**
Things this PR changes:
- replaced using deprecated url.parse method with URL
- fixed handler not returning any response
- code cleaned up a bit

### CODE REVIEW

**Comment by itsmingjie** [comment]
Thanks for this! This is actually extremely critical because if you were like me and just implemented it verbatim... You're gonna get hellofa Vercel usage report like this one...

Usage – Dashboard – Vercel - Arc - 2022-11-29 at 15 43 39@2x

File: tunneling/nextjs/pages/api/tunnel.js [hunk 1]
```diff
@@ -12,25 +11,23 @@ async function handler(req, res) {
...
-    const response = await fetch(url, {
-      method: "POST",
-      body: envelope,
-    });
-    return response.json();
+    const url = `https://${sentryHost}/api/${projectId}/envelope/`;    
+    const response = await fetch(url, { method: "POST", body: envelope });
+    const data = await response.json();
+
+    return res.status(200).json(data);
```
Comment (lobsterkatie) [inline tunneling/nextjs/pages/api/tunnel.js:30]:
```suggestion
    const sentryIngestURL = `https://${sentryHost}/api/${projectId}/envelope/`;
    const sentryResponse = await fetch(sentryIngestURL, { method: "POST", body: envelope });
    
    // Relay response from Sentry servers to front end
    sentryResponse.headers.forEach([key, value] => res.setHeader(key, value));
    res.status(sentryResponse.status).send(sentryResponse.body);
```

File: tunneling/nextjs/pages/api/tunnel.js [hunk 2]
```diff
@@ -12,25 +11,23 @@ async function handler(req, res) {
   try {
     const envelope = req.body;
     const pieces = envelope.split("\n");
-
     const header = JSON.parse(pieces[0]);
-
-    const { host, path } = url.parse(header.dsn);
+    const { host, pathname } = new URL(header.dsn);
+    const projectId = pathname.substring(1);
```
Comment (lobsterkatie) [inline tunneling/nextjs/pages/api/tunnel.js:16]:
```suggestion
    // DSNs are of the form `https://<key>@o<orgId>.ingest.sentry.io/<projectId>`
    const { host, pathname } = new URL(header.dsn);
    // Remove leading slash
    const projectId = pathname.substring(1);
```

**Review by lobsterkatie — APPROVED** [review]
Thanks for the contribution, @fifi98 ! I made a few small edits, but otherwise looks good!

**Pull request merged**